modules/introduction/partials/new-features-76.adoc

```diff
@@ -21,3 +21,11 @@ See xref:rest-api:rest-setting-security.adoc[Configure On-the-Wire Security].
 * Credentials for Couchbase-Server internal users can now be rotated at any time, by means of the REST API.
 See xref:rest-api:rest-rotate-internal-credentials.adoc[Rotate Internal Credentials].
 
+* One or more _Serviceless Nodes_ can now be added to a cluster.
```

Comment:
This should change to Arbiter Node

Reply:
(Disclaimer: I've not read the doc proposal in it's entirety yet, this comment is just about naming of this new capability)

I agree with Tony's comments in the email that @stevewatanabe shared that "Arbiter Node" isn't a good description of this feature. Rather than describe a new capability of the server (that we can add nodes running only the cluster manager) it describes one use-case for those nodes (acting as an arbiter). Given that we've identified at least two uses for these types of nodes so far (acting as an arbiter and for preferential orchestrator selection) I don't think that it's a particularly good idea to name them after one use case, rather than the capability that we've added. We don't know what other use cases we might find for these in the future, and "Arbiter Node" might not fit those particularly well.

A couple of points additionally worth highlighting:
1. In addition to the faster failover case, the preferential orchestrator selection can additionally be used to take orchestrator burden (the increased CPU/memory required) away from any other node.
1. Similarly to the example given [here](https://github.com/couchbase/docs-server/pull/3378#discussion_r1476127987), that any non-KV node can be used to the same extent as a service-less node to achieve faster failover, any node running any service can act as an arbiter to achieve a quorum majority. We've recommened, and I believe seen a few customer use, a backup service node for this in the past.

I'm not a fan of naming things, but IMO we should call these something like "Cluster Manager Node" (perhaps "Cluster Manager Only Node"), and explicitly call out that other service nodes will pick up cluster management activites if such a node is used, and that this is perfectly fine for most use cases.

Reply:
I can't come up with a better term off the top of my head (and I haven't fully read the draft doc or the associated PRDs). At my previous company, we called a similar feature  "standby nodes."

But, assuming that "Arbiter Node" is mentioned either in the UI, the API, or some message someplace in the product, it's too late to try to find the perfect term. I'll go ahead and update this PR with the approved "arbiter Node" term.

Reply:
FWIW this feature is only available in the REST API, it is not in the UI, and it is configured as an absence of services when we set up a node. We have not called this an "arbiter node" anywhere within ns_server.

Reply:
I've gone ahead and swapped all instances of serviceless node with arbiter node.